ingestion/src/metadata/ingestion/source/dashboard/dashboard_source.py

```diff
@@ -80,18 +81,17 @@ def __init__(
         self.source_config: DashboardServiceMetadataPipeline = (
             self.config.sourceConfig.config
         )
-
         self.connection = get_connection(self.service_connection)
         self.client = self.connection.client
         self.service = self.metadata.get_service_or_create(
             entity=DashboardService, config=config
         )
-        self.status = SourceStatus()
+        self.status = SQLSourceStatus()
```

Comment:
Maybe not the best status to use here hehe Opened a ticket to track this https://github.com/open-metadata/OpenMetadata/issues/5247, let's move on for now